app/src/main/java/org/dhis2/usescases/main/program/ProgramFragment.kt

```diff
@@ -250,26 +250,23 @@ class ProgramFragment : FragmentGlobalAbstract(), ProgramView, OnOrgUnitSelectio
     }
 
     override fun showSyncDialog(program: ProgramViewModel) {
-        val dialog = SyncStatusDialog.Builder()
-            .setConflictType(
-                if (program.programType.isNotEmpty()) {
-                    ConflictType.PROGRAM
-                } else {
-                    ConflictType.DATA_SET
+        SyncStatusDialog.Builder()
+            .withContext(this)
+            .withSyncContext(
+                when (program.programType) {
+                    "WITH_REGISTRATION" -> SyncContext.GlobalTrackerProgram(program.uid)
```

Comment:
There is not enum items for programType instead of hardcoded text?

Reply:
☝🏽